src/styles/StyleUtils.js

```diff
@@ -393,6 +393,17 @@ function parseStyleAsArray(styleParam) {
     return _.isArray(styleParam) ? styleParam : [styleParam];
 }
 
+/**
+ * Return LoginField label margin.
+ *
+ * @param {Boolean} addedPhone
+ * @param {Boolean} validatedPhone
+ * @returns {Object}
+ */
+function getLoginFieldLabelMargin(addedPhone, validatedPhone) {
```

Comment:
@thesahindia Really small preference, but let's put the function in loginfield.js?
Thanks for bearing with me